staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics/metrics.go

```diff
@@ -146,6 +146,18 @@ var (
 		},
 		[]string{"resource"},
 	)
+
+	WatchListLatencies = compbasemetrics.NewHistogramVec(
```

Comment:
@dgrisonnet could please take a look at this metric ?

My intention here is to measure latency of a special type of watch requests called `watch-list`. 
I'd like to compare it against standard list request to see how the new mechanism performs.

The standard list requests are measured by https://github.com/kubernetes/kubernetes/blob/c607984379234249403354bf1f9a2e86a740caf0/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L91

I'm not sure if the `Buckets` need to match or not.

Reply:
I opened https://github.com/kubernetes/kubernetes/pull/120520 as it will make the logic that determines the scope of a request easier.

Reply:
I've also opened https://github.com/kubernetes/perf-tests/pull/2315 that will allow us to measure the new metric in the perf test